futures-util/src/try_stream/mod.rs

```diff
@@ -262,4 +292,5 @@ pub trait TryStreamExt: TryStream {
     {
         TryFilterMap::new(self, f)
     }
+
```

Comment:
We usually don't have padding in impl blocks.

Reply:
Added this by accident, done